test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultEditorTemplatesTest.cs

```diff
@@ -28,48 +28,56 @@ public class DefaultEditorTemplatesTest
             {
                 return new TheoryData<string, string>
                 {
-                    { null, "__TextBox__" },
-                    { string.Empty, "__TextBox__" },
-                    { "EmailAddress", "__TextBox__" },
-                    { "emailaddress", "__TextBox__" },
+                    { null, "__TextBox__ class='text-box single-line'" },
```

Comment:
no.  `StubbyHtmlHelper` previously ignored the passed `htmlAttributes`.  added that so the chosen `class` and `type` values come through.

the `format` fix led only to the one line functional test change